core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/codesystem/CodeSystemVersion.java

```diff
@@ -95,36 +118,29 @@ public void setDescription(final String description) {
 		this.description = description;
 	}
 
+	public void setPath(String path) {
+		this.path = path;
+	}
+	
+	@Deprecated
+	public void setRepositoryId(String repositoryId) {
+		this.repositoryId = repositoryId;
+	}
+	
 	public void setVersion(final String version) {
 		this.version = version;
 	}
 	
-	public void setParentBranchPath(final String parentBranchPath) {
-		this.parentBranchPath = parentBranchPath;
+	public void setUri(CodeSystemURI uri) {
+		this.uri = uri;
 	}
-
-	public void setPatched(final boolean patched) {
-		this.patched = patched;
+	
+	/**
+	 * @return {@code true} if this version represents the HEAD in the repository.
+	 */
+	@JsonIgnore
+	public boolean isLatestVersion() {
+		return MAIN_BRANCH.equals(getVersion());
```

Comment:
Where do we build an artificial "latest" version like this? I couldn't find references in Eclipse for it.

Reply:
In other projects. This method is currently referenced in this repository.